samples/net/wpanusb/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _wpanusb-sample:
+
+WPANUSB sample
+##############
+
+Overview
+********
+
+This application exports ieee802154 radio over USB to be used in other
+Operation Systems like Linux. In this scenario Linux SoftMAC driver would
+be used implementing ieee802154 stack inside Linux.
+
+Requirements
+************
+
+- Linux machine host connected via USB to Zephyr WPANUSB enabled MCU
```

Comment:
"Linux machine host" sounds a bit artificial. Maybe just "Linux host"?

Reply:
"Zephyr WPANUSB enabled MCU" sounds strange. Maybe "Zephyr WPANUSB enabled board"?

Reply:
Also, we usually consider smaller boards to be connected to a host, not vice-versa ;-).